docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - `Improvements` - Unuseful log about missed i18n section has been removed  [#1269](https://github.com/codex-team/editor.js/issues/1269)
 - `Fix` - Fixed issue with enter key in inputs and textareas [#920](https://github.com/codex-team/editor.js/issues/920)
 - `Improvements` - Allowed to set `false` as `toolbox` config in order to hide Toolbox button [#1221](https://github.com/codex-team/editor.js/issues/1221)
+- `Fix` - Fixed issue with getBlockByIndex API [#1270](https://github.com/codex-team/editor.js/issues/1270)
```

Comment:
what issue?

Reply:
It's a bug which will return the block object when no data present in EditorJS holder and we call editor.getBlockByIndex(1).